pyuvdata/uvflag/uvflag.py

```diff
@@ -2074,8 +2077,26 @@ def _select_preprocess(
                     "(optionally with polarization)."
                 )
             if not all(
-                [isinstance(item[0], (int, np.integer,)) for item in bls]
-                + [isinstance(item[1], (int, np.integer,)) for item in bls]
+                [
+                    isinstance(
+                        item[0],
+                        (
+                            int,
+                            np.integer,
+                        ),
+                    )
+                    for item in bls
+                ]
+                + [
+                    isinstance(
+                        item[1],
+                        (
+                            int,
+                            np.integer,
+                        ),
+                    )
+                    for item in bls
+                ]
```

Comment:
Another candidate for refactoring. I find this painful to try to parse.

Reply:
Current versions of `black` will not expand this as long as it fits on one line and the arguments don't end with a trailing comma.

Reply:
I think the change is caused by an upgrade to Black, from 19.10b0 to 22.1.0.

Reply:
Fixed in b5ea2cbd.